### DISCUSSION TITLE
I have no way to try the Copilot out.

### DISCUSSION THREAD

**Discussion by cyanzhong** [community/community#19119] [Copilot Conversations] [answered]
🎉 Congrats to the GitHub Copilot's GA! I was about to try it out and met some issues.

Right after clicking the `Start my free trial` button (https://github.com/features/copilot/):

<img width="640" alt="image" src="https://user-images.githubusercontent.com/6745066/174934215-71499f23-b4c0-42b7-bb47-bcd07926cd18.png">

I will be redirected to the page for setting up billing information (https://github.com/settings/billing/, and unfortunately on this page I don't see an option for Copilot), I have my billing info set up, so tbh I don't know what's the next step for me from there.

Anyone else seeing the same? Thanks.

**Comment by gup97** [comment]
an answer to `Anyone else seeing the same?`

Right after clicking the `Start my free trial`

In my case it's to go to this  [link](https://github.com/github-copilot/signup)
link : https://github.com/github-copilot/signup

![image](https://user-images.githubusercontent.com/80537765/174936565-64deccbf-e46f-4160-8f70-c59152378b38.png)

Right after clicking, [  Continue to get access to Copilot](https://github.com/github-copilot/signup/billing)

![image](https://user-images.githubusercontent.com/80537765/174936751-da21589d-dbd2-42ca-9e92-3b9f3226c82a.png)

Is that the answer you want? If not, I'll fix it 😐

**Reply by cyanzhong** [reply]
@gup97 Thanks for answering, mate. Unfortunately, both links you mentioned keep sending me to the billing page I mentioned above.

Seems like something idk is wrong.

**Reply by gup97** [reply]
Have you tried it?  in https://github.com/settings/billing/
![image](https://user-images.githubusercontent.com/80537765/174937549-502e2c96-9d3f-41b3-890a-a08082198572.png)
![image](https://user-images.githubusercontent.com/80537765/174937690-9bde8eee-4b66-48b1-bfd4-7f631fbd7586.png)

**Reply by cyanzhong** [reply]
The thing is, on this page I don't see an option for Copilot as the screenshot you shared, weird.

**Reply by gup97** [reply]
[I looked it up and found out that many people are experiencing it](https://github.com/orgs/github-community/discussions/19103)

But I can't help you Sorry...

**Reply by cyanzhong** [reply]
Thanks anyway! It does help since I am not the only one.

**Comment by davecheney** [comment, marked as answer]
@cyanzhong you may be receiving copilot access via a trial from your employer. Is copilot working for you at the moment?

**Reply by cyanzhong** [reply]
Thanks for the reply, I don't quite get it. How would my employer offer me a free trial? To answer your question, no, it's not working.

**Reply by davecheney** [reply]
Thanks for confirming. It was a long shot. I suggest contacting support who can confirm your billing and account information is correct.

**Reply by cyanzhong** [reply]
I think you were right, I was eventually offered access by my employer, Microsoft.

**Reply by PreetM07** [reply]
i have not offered access by my employer! Help
